cogs/economy.py

```diff
@@ -123,6 +123,7 @@ def create_account(self, user, *, initial_balance=0):
                 self.accounts[server.id] = {}
             if user.id in self.accounts:  # Legacy account
                 balance = self.accounts[user.id]["balance"]
+                int(balance)
```

Comment:
This has no effect: `balance` is still a double/float.  Let's remove it, and we can fix the accounts in the data file itself afterwards.

Reply:
Alternatively, you could cast the entire `self.accounts[user.id]["balance"]` to an int, but I don't think we need to per se.